projects/ccf-rui/src/app/modules/left-sidebar/left-sidebar.component.html

```diff
@@ -11,16 +11,16 @@
     <ccf-labeled-slide-toggle class="toggle gender" [labels]="['Male', 'Female']"></ccf-labeled-slide-toggle>
   </div>
   <hr>
-  <mat-expansion-panel class="extraction-menu">
+  <mat-expansion-panel (opened)="load = true" [class.load]="load" class="extraction-menu">
     <mat-expansion-panel-header class="expansion-header">
       <mat-panel-title class="expansion-title">Common Extraction Sites</mat-panel-title>
     </mat-expansion-panel-header>
-    <ccf-visibility-menu [items]="extractionSites"></ccf-visibility-menu>
+    <ccf-visibility-menu  [items]="extractionSites"></ccf-visibility-menu>
   </mat-expansion-panel>
   <hr>
   <ccf-visibility-toggle [toggleLabel]="'Show Previous Registration Blocks'"></ccf-visibility-toggle>
   <hr>
-  <mat-expansion-panel class="anatomical-structures-menu">
+  <mat-expansion-panel (opened)="load = true" [class.load]="load" class="anatomical-structures-menu">
```

Comment:
Why not use the expanded property??? https://material.angular.io/components/expansion/api

Reply:
And/or lazy-rendering: https://material.angular.io/components/expansion/overview#lazy-rendering

Reply:
I tried that but the contents were showing even when expanded was set to false. Didn't see the lazy rendering thing though, I can see if that works.

Reply:
So it's a bit better now but still a little weird. Now instead of the content there's just an empty space that disappears after a second. Setting expanded to false doesn't do anything.

Reply:
I can have a look at this if it would help?

Reply:
Sure